enzy_htp/structure/residue.py

```diff
@@ -55,7 +55,15 @@ def atoms(self) -> List[Atom]:
     @atoms.setter
     def atoms(self, val):
         self.set_children(val)
-
+    
+    @property
+    def _atoms(self) -> List[Atom]:
+        """alias for _children. Prevent change children but _atoms dont change"""
+        return self._children
+    @_atoms.setter
```

Comment:
Add a space here between this and the above method.

Reply:
Not having space here was because those getter/setter methods are meant to be in pair. I see treat them here together as a "method".